### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by imjorge**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+To install:
+
+```
+$ go get github.com/flynn/flynn-discovery
+$ cd $GOPATH/src/github.com/flynn/flynn-discovery
+$ go build
+```
+
+`flynn-discovery` requires a Postgres database to save cluster information. For this demo we will use a Docker container.
```
Comment (lmars) [inline README.md]: How about deploy it to a Flynn cluster which has Postgres already running :wink:

Comment (imjorge) [inline README.md]: Added.

**Comment by lmars** [comment]
@imjorge please add a [DCO sign off](https://flynn.io/docs/contributing#developer’s-certificate-of-origin) to the commits.

**Comment by titanous** [comment]
We should drop the Docker example.

**Comment by imjorge** [comment]
Removed Docker example and added DCO sign off. Although I think it would be useful for using `flynn-discovery` outside of Flynn which embodies the Flynn mantra of modularity where one could use components from it in isolation.

**Comment by lmars** [comment]
@imjorge you need to add the DCO sign-off to each of the commits (you may as well squash them into one at the same time)

**Comment by imjorge** [comment]
@lmars done.

**Comment by lmars** [comment]
LGTM

**Comment by lmars** [comment]
Thanks!

**Pull request merged**